virtru-audit-client/utils.py

```diff
@@ -0,0 +1,92 @@
+import configparser
+import datetime
+import os
+import json
+import sys
+import csv
+import pathlib
+import logging
+from enum import Enum
+from logging.handlers import SysLogHandler
+from rfc5424logging import Rfc5424SysLogHandler
+
+BOOK_MARK_FILE_NAME = 'bookmark.ini'
+
+
+class AuditTypes(Enum):
+    API_TOKEN = 'api-token'
+    APP_ID_BUNDLE = 'appIdBundle'
+    CONTRACT_GET = 'contract-get'
+    DLP_RULES = 'dlp-rules',
+    DLP_OVERRIDE = 'dlpOverride'
+    ENCRYPTED_SEARCH_KEY = 'encrypted-search-key'
+    LICENSE_INVITATION = 'licenseInvitation'
+    ORGANIZATION = 'organization'
+    POLICY = 'policy'
+    UNIT_ATTRIBUTES = 'unit-attributes'
+    USER_SETTINGS = 'userSettings'
+
+
+def getConfig(configFile):
+    try:
+        config = configparser.ConfigParser()
+        with open(configFile) as f:
+            config.read_file(f)
+        return config['ApiInfo']
+    except FileNotFoundError as err:
+        logging.error(err)
+        sys.exit(-1)
+
+
+def getNextPageStartKey():
+    bookmark = configparser.ConfigParser()
+    bookmark.read(BOOK_MARK_FILE_NAME)
+    if len(bookmark) <= 1:
+        return None
+    else:
+        return bookmark['next-page-start-key']
+
+
+def saveNextPgeStartKey(nextPageStartKey):
+    bookMarkConfig = configparser.ConfigParser()
+    bookMarkConfig['next-page-start-key'] = {
+        'nextPageStartKey': nextPageStartKey}
+    with open(BOOK_MARK_FILE_NAME, 'w') as bookMarkFile:
+        bookMarkConfig.write(bookMarkFile)
+
+
+def exportToJson(pathToFolder, records):
+
+    fileName = str(datetime.datetime.utcnow().isoformat()) + ".json"
+    fn = os.path.join(pathToFolder, fileName)
+    with open(fn, "w") as f:
```

Comment:
(super) nit: double quotes here but you usually have used single

Reply:
I guess I should stick to one style